### DISCUSSION TITLE
NAND access and boot from SATA

### DISCUSSION THREAD

**Discussion by bethencsoft** [johang/sd-card-images#48] [Q&A] [answered]
Hi,

Please, sorry my poor English and my little knowledge about this.

I have an allwinner-a10 (mele a1000) resurrected by your amazing job. Thanks for this.

Somebody knows if would be possible NAND access and booting from SATA or where start to investigate about this?

All information about this I found in google is too old and links are broken.

I can't find nanda and nandb in /dev

Can I help me about this?

Thanks.

**Comment by johang** [comment, marked as answer]
See #9.

**Reply by bethencsoft** [reply]
Thanks!
